sound/soc/intel/boards/sof_sdw_maxim.c

```diff
@@ -149,6 +158,9 @@ int sof_sdw_maxim_init(struct snd_soc_card *card,
 
 	maxim_part_id = info->part_id;
 	switch (maxim_part_id) {
+	case SOF_SDW_PART_ID_MAX98363:
+		dai_links->ops = &max_98363_sdw_ops;
```

Comment:
see comment above, I think the ops are already set. Can you double-check?

Reply:
Done the changes